learn/core_concepts/documents.md

```diff
@@ -54,12 +54,35 @@ An attribute functions a bit like a variable in most programming languages, i.e.
 
 Every field has a [data type](/reference/under_the_hood/datatypes.md) dictated by its value. Every value must be a valid [`JSON` data type](https://www.w3schools.com/js/js_json_datatypes.asp).
 
-Take note that in the case of strings, the value **[can contain at most 1000 words](/reference/features/known_limitations.md#maximum-words-per-attribute)**. If it contains more than 1000 words, only the first 1000 will be indexed.
-
 You can also apply [<clientGlossary word="ranking rules" />](/learn/core_concepts/relevancy.md#ranking-rules) to some fields. For example, you may decide recent movies should be more relevant than older ones.
 
 If you would like to adjust how a field gets handled by MeiliSearch, you can do so in the [settings](/reference/features/settings.md#settings).
 
+Take note that in the case of strings, the value **[can contain at most 1000 positions](/reference/features/known_limitations.md#maximum-words-per-attribute)**. Words exceeding the 1000 position limit will be ignored.
+
+### Example
+
+If your query is `Hello World`
+
+- `Hello` takes the position `0` of the attribute
```

Comment:
It seems correct to me to speak of attribute in this case. In a more general way to join what @curquiza said, I am afraid that it adds complexity to go into details about the limit.

We are working on removing this limit for Q4. Have you had a lot of feedback about this limit (I've seen it a few times, especially because users have to split one attribute into multiple attributes to overcome this limit)? Does the explanation make a difference to the user?

Reply:
I agree with @gmourier, this brings complexity in this first approach of the MeiliSerch documents